src/types/rest/response/account.ts

```diff
@@ -0,0 +1,68 @@
+export interface Position {
+  cost: number;
+  cumulativeBuySize: number;
+  cumulativeSellSize: number;
+  entryPrice: number;
+  estimatedLiquidationPrice: number;
+  future: string;
+  initialMarginRequirement: number;
+  longOrderSize: number;
+  maintenanceMarginRequirement: number;
+  netSize: number;
+  openSize: number;
+  realizedPnl: number;
+  recentAverageOpenPrice: number;
+  recentBreakEvenPrice: number;
+  recentPnl: number;
+  shortOrderSize: number;
+  side: string;
+  size: number;
+  unrealizedPnl: number;
+  collateralUsed: number;
+}
+
+export type Positions = Array<Position>
+export interface AccountSummary {
+  accountIdentifier: number;
+  username: string;
+  collateral: number;
+  freeCollateral: number;
+  totalAccountValue: number;
+  totalPositionSize: number;
+  initialMarginRequirement: number;
+  maintenanceMarginRequirement: number;
+  marginFraction: number;
+  openMarginFraction: number;
+  liquidating: boolean;
+  backstopProvider: boolean;
+  positions: Positions;
```

Comment:
Nit: would much prefer avoiding "plural" variations of types, just use array notation anywhere it's an array of an existing type/interface. Example:
```suggestion
  positions: Position[];
```